dingomata/cogs/random_response_data.yaml

```diff
@@ -22,6 +22,15 @@ pour.tea:
   fragments:
     action: [ pours, gives, hands, offers, passes, orders ]
     unit: [ cup, mug, shot, pot, gallon, quart, pint ]
+pour.orangina:
+  templates:
+    - '{author} politely {action} {target} a {unit} of Sparkling Orangina. Mmmmm, Citrusy!'
+    - '{author} hands {target} a glass of Orangina and a tangerine... or is it a grapefruit?'
+    - '{author} suddenly transports {target} into an Orangina ad. You're on TV now!'
```

Comment:
You'll need to escape the single quote inside or use double quotes instead :3c

Reply:
Yep, and that is why I shouldn't write code while playing Risk of Rain at the same time.  I'll fix that tonight.